paymentsheet/src/main/java/com/stripe/android/customersheet/util/CustomerSheetHacks.kt

```diff
@@ -23,7 +25,7 @@ import kotlinx.coroutines.flow.first
  * This objects holds references to objects that need to be shared across Activity boundaries
  * but can't be serialized, or objects that can't be injected where they are used.
  */
-@OptIn(ExperimentalCustomerSheetApi::class)
+@OptIn(ExperimentalCustomerSheetApi::class, ExperimentalCustomerSessionApi::class)
 internal object CustomerSheetHacks {
```

Comment:
Not for this PR ofc, but I feel like we should rename this if it's going to continue to be used for managing the data sources. It's stressing me out that we have such a big + important object named "Hacks" lol

Reply:
I agree!

Reply:
ty!